crowbar_framework/app/models/service_object.rb

```diff
@@ -292,7 +292,8 @@ def versions
     [200, { versions: ["1.0"] }]
   end
 
-  def transition
+  def transition(inst, name, state)
+    Rails.logger.debug("transition: enter #{inst} #{name} #{state}")
```

Comment:
Please don't add just a "enter" log line, as we'll be looking for the "leave" log line. Can you either drop the verb, or make it "enter and leave"?